pkgs/development/python-modules/kikit/default.nix

```diff
@@ -0,0 +1,87 @@
+{ bc
+, zip
+, lib
+, fetchFromGitHub
+, bats
+, buildPythonPackage
+, pythonOlder
+, callPackage
+, kicad
+, numpy
+, click
+, markdown2
+, commentjson
+, wxPython_4_2
+, pcbnew-transition
+, pybars3
+, pytest
+, versioneer
+}:
+let
+  properCaseName = "KiKit";
+  solidpython = callPackage ./solidpython { };
+
+  # https://github.com/yaqwsx/KiKit/issues/574
+  # copy-pasted from nixpkgs#8d8e62e74f511160a599471549a98bc9e4f4818d
+  shapely = callPackage ./shapely { };
```

Comment:
This should be added to pythonPackages instead

Reply:
shapely is [already in nixpkgs](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/shapely/default.nix). We need a much older version here. Does this address your concern, please?

By the way, upstream is [planning to drop shapely](https://github.com/yaqwsx/KiKit/issues/574#issuecomment-1657787964), if that matters.

Reply:
Marking this resolved. Please reopen if you'd like us to do something different here.

Reply:
> shapely is [already in nixpkgs](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/shapely/default.nix?rgh-link-date=2023-08-28T06%3A28%3A43Z). We need a much older version here. Does this address your concern, please?

Then kitkit needs to move out of pythonPackages.